src/components/button/styles.ts

```diff
@@ -10,6 +10,7 @@ export const elButtonLabel = css``
 export const elIcon = css``
 export const elButtonIconOnly = css``
 export const elButtonSpinner = css``
+export const elButtonRemovePadding = css``
```

Comment:
**suggestion:** I think a data attribute is a nicer, more "componenty" approach than class names. For example, if the React component prop is `hasNoPadding` then the styled component here would accept `data-has-no-padding` and the button styles would use `&[data-has-no-padding="true"] { ... }`

Reply:
Thanks for the reminder, I always forget this one.